### PR TITLE
Azure - Node - Service Fabric Code Updates

### PR DESCRIPTION
**Pull request by jawache**
Updated code example for service fabric

#sign-off as per @TomArcher

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/service-fabric.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-5/docs-ref-services/service-fabric.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C13%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707130854417479-78%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/service-fabric.md [hunk 1]
```diff
@@ -17,20 +17,38 @@ ms.service: Service Fabric
...
-### Management
+### Install npm module
 ```bash
 npm install azure-arm-servicefabric
 ```
 
 
-##Example
+### Example
+```javascript
```
Comment (johnpapa) [inline docs-ref-services/service-fabric.md]: add a brief phrase that explains what the example does

File: docs-ref-services/service-fabric.md [hunk 2]
```diff
@@ -17,20 +17,38 @@ ms.service: Service Fabric
 
 ## Overview
 
-## Install the modules with npm
+## Management Package
 
 Use npm to install the Azure Service Fabric modules for Node.js
```
Comment (johnpapa) [inline docs-ref-services/service-fabric.md]: modules --> module

since there is only one here

**Comment by TomArcher** [comment]
@jawache Conflicts. Please resolve

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/service-fabric.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-5/docs-ref-services/service-fabric.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C13%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707131431056101-78%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by TomArcher** [comment]
@jawache Merged

**Pull request merged**